TestCases/cont_adj_euler/oneram6/inv_ONERAM6.cfg

```diff
@@ -369,14 +369,12 @@ FFD_CONTINUITY= 2ND_DERIVATIVE
 %    FREE_SURFACE, AVG_TOTAL_PRESSURE, MASS_FLOW_RATE
 %
 % Available geometrical based objective functions or constraint functions
-%    MAX_THICKNESS, 1/4_THICKNESS, 1/2_THICKNESS, 3/4_THICKNESS, AREA, AOA, CHORD, 
-%    MAX_THICKNESS_SEC1, MAX_THICKNESS_SEC2, MAX_THICKNESS_SEC3, MAX_THICKNESS_SEC4, MAX_THICKNESS_SEC5, 
-%    1/4_THICKNESS_SEC1, 1/4_THICKNESS_SEC2, 1/4_THICKNESS_SEC3, 1/4_THICKNESS_SEC4, 1/4_THICKNESS_SEC5, 
-%    1/2_THICKNESS_SEC1, 1/2_THICKNESS_SEC2, 1/2_THICKNESS_SEC3, 1/2_THICKNESS_SEC4, 1/2_THICKNESS_SEC5, 
-%    3/4_THICKNESS_SEC1, 3/4_THICKNESS_SEC2, 3/4_THICKNESS_SEC3, 3/4_THICKNESS_SEC4, 3/4_THICKNESS_SEC5, 
-%    AREA_SEC1, AREA_SEC2, AREA_SEC3, AREA_SEC4, AREA_SEC5, 
-%    AOA_SEC1, AOA_SEC2, AOA_SEC3, AOA_SEC4, AOA_SEC5, 
-%    CHORD_SEC1, CHORD_SEC2, CHORD_SEC3, CHORD_SEC4, CHORD_SEC5
+%    AIRFOIL_AREA, AIRFOIL_THICKNESS, AIRFOIL_CHORD, AIRFOIL_TOC, AIRFOIL_AOA,
+%    WING_VOLUME, WING_MIN_THICKNESS, WING_MAX_THICKNESS, WING_MAX_CHORD, WING_MIN_TOC, WING_MAX_TWIST, WING_MAX_CURVATURE, WING_MAX_DIHEDRAL
+%    FUSELAGE_VOLUME, FUSELAGE_WETTED_AREA, FUSELAGE_MIN_WIDTH, FUSELAGE_MAX_WIDTH, FUSELAGE_MIN_HEIGHT, FUSELAGE_MIN_HEIGHT, FUSELAGE_MAX_CURVATURE
+%    STATION#_WIDTH, STATION#_WATERLINE_WIDTH, STATION#_HEIGHT,
```

Comment:
# is the index of the station defined in GEO_LOCATION_STATIONS I have updated the config file with that extra info. Thanks!